packages/amplify-provider-awscloudformation/lib/push-resources.js

```diff
@@ -474,6 +484,46 @@ function updateIdPRolesInNestedStack(context, nestedStack, authResourceName) {
 
   Object.assign(nestedStack.Resources, idpUpdateRoleCfn);
 }
+ function packageLayer(context,resource){
+  const resourcePath = path.join(context.amplify.pathManager.getBackendDirPath(), resource.category, resource.resourceName);
+  //const resourcePathSrc = path.join(resourcePath,'src');
+  const layerObj = context.amplify.readJsonFile(path.join(resourcePath,'layer-parameters.json'));
+  const layerVersionArn = layerObj.parameters.layerVersionsArray[0];
+  const zipFilename = 'latest-build.zip';
+
+  // package the function
+  const distDir = path.join(resourcePath, 'dist');
+  if (!fs.existsSync(distDir)) {
+    fs.mkdirSync(distDir);
+  }
+  const destination = path.join(distDir,zipFilename );
+  // verify the folder structure
```

Comment:
nit: remove comment since we decided to not verify structure in case user wants to configure their own custom paths in the `/opt` folder

Reply:
done